### DISCUSSION TITLE
QSV HW Rescaler not supported?

### DISCUSSION THREAD

**Discussion by zapf2000** [OvenMediaLabs/OvenMediaEngine#1475] [Help]
I can't find anything that qsv hardware rescaler is supported in OME. For nvidia it's there. qsv encoding and decoding works fine. 
So why not add the scale_qsv filter to prerequisites.sh and add the correct syntax (for example call ffmpeg -y -init_hw_device qsv=hw,child_device_type=dxva2 -hwaccel qsv -hwaccel_output_format qsv -c:v h264_qsv -i input.mp4 -vf "vpp_qsv=framerate=60,scale_qsv=w=640:h=480" -c:v h264_qsv output.mp4) in filter_rescaler.cpp ?

**Comment by getroot** [comment]
Since there seemed to be no users using QSV, it was pushed down the priority list. I think most of them are using NVIDIA or Xilinx. Anyway, this is in our backlog and we will increase the priority as we get more requests from users.

**Reply by zapf2000** [reply]
It makes absolute sense, since there are a lot of people use dedicated servers with an Intel CPU and a lot of them have a GPU on chip (like in my case). In the future you also plan to support AV1 and the cheapest HW solution for that is to buy an Intel ARC GPU.  I already run it sucessfully on Ubuntu (makes 6 30fps 1080p30 AV1 in parallel).
